src/scss/main.scss

```diff
@@ -61,11 +62,6 @@
 @import '../components/timeline/timeline';
 @import '../components/upload/upload';
 @import '../components/video/video';
-@import '../patterns/help-users-to/see-important-information/src/_highlight';
```

Comment:
No really my place to comment but `@import '../components/highlight/highlight';` on L:31 tells me that there's a big chance `../patterns/help-users-to/see-important-information/` might be missing from `src/scss/patternlib.scss`.